include/Snake.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef SNAKE_H
+#define SNAKE_H
+
+#include <cassert>
+#include <vector>
+
+#include "../Util/Vector2D.h"
+#include "../settings.h"
+
+class Snake
+{
+public:
+    enum class Direction
+    {
+        up,
+        down,
+        left,
+        right,
+        max_directions
+    };
+
+    explicit Snake(int x, int y, int length, Direction dir);
```

Comment:
use Vector2D instead of two variables x and y

Reply:
also i think we don't need length as an argument because by default snake has no length